lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java

```diff
@@ -54,7 +59,15 @@ private static FormatterStep create(String version, Provisioner provisioner, boo
 		Objects.requireNonNull(version, "version");
```

Comment:
You can replace this whole body with just `return create(version, provisioner, isScript, Collections.emptyMap())`.